CHANGELOG/CHANGELOG-2.0.md

```diff
@@ -1,3 +1,50 @@
+# Release notes for v2.0.5
+
+[Documentation](https://docs.k8s.io/docs/home)
+# Changelog since v2.0.4
+
+## Changes by Kind
+
+### Bug or Regression
+
+- `k8s.io/csi-translation-lib` dependency is updated from v0.19.3 to v0.19.9 to adopt fixes in the upstream library. ([#572](https://github.com/kubernetes-csi/external-provisioner/pull/572), [@ialidzhikov](https://github.com/ialidzhikov))
```

Comment:
Are there any other updates we made through 1.21 that we want to include here? cc @Jiawei0227

Reply:
Do you mean we want to cherry pick some of the csi migration fix to the provisioner 2.0 release branch, mainly topology ones? I guess it is okay we not include them since those will be included in the new csi provisioner release